CalPatRec/src/CalTimePeakFinder_module.cc

```diff
@@ -200,7 +200,7 @@ namespace mu2e {
     static const double twopi(2*pi);
 
     double              mphi(-9999.);
-    double              meanDriftTime = 1.25/0.06;// half straw tube radius / drift velocity
+    double              meanDriftTime = -3.3;//1.25/0.06;// half straw tube radius / drift velocity
```

Comment:
What does this number correspond to?  If it depends on gas properties it should come from StrawResponse.

Reply:
Waiting for @bonventre to comment on this

Reply:
The TOT calibration currently has a 3 ns bias (needs to be recalibrated based on MDC2020 samples), so this is expected right now but should be closer to zero once that's fixed. After recalibrating should check this against https://github.com/Mu2e/Offline/blob/main/TrkReco/inc/TrkTimeCalculator.hh#L37, maybe should be the same fcl parameter if there still ends up being an offset

Reply:
good point. I use now an fcl parameter that was already there.